var/spack/repos/builtin/packages/umpire/package.py

```diff
@@ -16,14 +16,27 @@ class Umpire(CMakePackage):
 
     version('develop', branch='develop', submodules='True')
     version('master', branch='master', submodules='True')
-    version('0.1.4', tag='v0.1.4', submodules='True')
     version('0.1.3', tag='v0.1.3', submodules='True')
+    version('0.1.4', tag='v0.1.4', submodules='True')
+    version('0.2.0', tag='v0.2.0', submodules='True')
+    version('0.2.1', tag='v0.2.1', submodules='True')
+    version('0.2.2', tag='v0.2.2', submodules='True')
+    version('0.2.3', tag='v0.2.3', submodules='True')
+    version('0.2.4', tag='v0.2.4', submodules='True')
+    version('0.3.0', tag='v0.3.0', submodules='True')
+    version('0.3.1', tag='v0.3.1', submodules='True')
+    version('0.3.2', tag='v0.3.2', submodules='True')
+    version('0.3.3', tag='v0.3.3', submodules='True')
```

Comment:
Usually we put new versions at the top, so the order is the reverse of this one. Can you do the same above?

Reply:
Done.